pkg/model/serializer.go

```diff
@@ -241,20 +241,22 @@ func (e Entrypoint) MarshalYAML() (interface{}, error) {
 func (c *Command) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	var multi []string
 	err := unmarshal(&multi)
-	if err != nil {
```

Comment:
I think we should do the change of the serializer in a different PR so that we can keep the scope as small as possible, it's a change required here but also in the [PR](https://github.com/okteto/okteto/pull/3833). for refactoring the hybrid executors. It would also be better from a git history point of view. WDYT?

Reply:
I agree but when we do that the issue will be resolved. Should we change the PR to remove the wrapper in the serializer and fix the hybrid working different as a side effect?

Reply:
Included it on the title to make it more visible. Let me know if that's enough